cmd/secret-wrapper/main.go

```diff
@@ -172,6 +172,18 @@ func execCmd(argv []string) error {
 	proc := exec.Command(argv[0], argv[1:]...)
 	proc.Stdout = os.Stdout
 	proc.Stderr = os.Stderr
+	// provide a writeable home so kubectl discovery can be cached
```

Comment:
Can we rename the tool if it now does stuff that is entirely unrelated to secret wrapping?

Reply:
Sure, can I do that after I land the useful bits? :)

Reply:
I am afraid that won't happen then :P Test failure looks genuine btw